run-media-source-web-platform.sh

```diff
@@ -15,10 +15,38 @@ PLATFORM=$3
 
 WORKSPACE=`pwd`
 
+function set_virtualenv {
+    # Try to use the virtual env that may have been activated
+    # before this script was called from cmd.exe; otherwise create a new one.
+    if [ -d "$VIRTUAL_ENV" ]; then
+        unset _OLD_VIRTUAL_PATH
```

Comment:
Need to unset `_OLD_VIRTUAL_PATH`, etc, because they refer to values that only make sense in the parent shell (e.g. `cmd.exe`) and we don't want the `activate` script to bring them back.